AlertaDengue/dbf/pysus.py

```diff
@@ -59,19 +61,15 @@
 while True:
     try:
         DISEASE = str(input("Digite a doença: "))
-        if (
-            DISEASE != "dengue"
-            and DISEASE != "chikungunya"
-            and DISEASE != "zika"
-        ):
+        if DISEASE not in CID10.keys():
             raise ValueError
```

Comment:
It's good practice to add a message when you raise an exception, so that the information about the error goes to the log file:
```python
raise ValueError(f"{DISEASE} is not a valid disease name")
```